keps/sig-apps/3335-statefulset-slice/README.md

```diff
@@ -1045,7 +1051,8 @@ Major milestones might include:
 
   - 2022-06-02: KEP created.
   - 2022-10-06: Alpha implementation.
-  - 2023-02-09: Beta implementation.
+  - 2023-02-09: Beta graduation.
```

Comment:
New question was added to the template in the meantime - please fill in:
https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md#can-enabling--using-this-feature-result-in-resource-exhaustion-of-some-node-resources-pids-sockets-inodes-etc

Reply:
Thanks. Added that question/answer here, and also addressed the new version skew question around `kube-controller-manager` https://github.com/kubernetes/enhancements/pull/4137